app/classes/Transvision/Project.php

```diff
@@ -169,6 +170,29 @@ public static function getRepositoryLocales($repository)
     }
 
     /**
+     * Get the list of repositories available for a locale
+     *
+     * @param  string $locale Mozilla locale code
+     * @return array  A sorted list of repositories available for the locale
+     */
+    public static function getLocaleRepositories($locale)
+    {
+        $matches = [];
+        foreach (self::getRepositories() as $repository) {
+            if (in_array(
+                self::getLocaleInContext($locale, $repository),
```

Comment:
You want `$locale` here, not the contextualized locale code. `/api/v1/repositories/es/`should return the locales actually supported by `es`: `["firefox_ios","gaia","gaia_1_3","gaia_1_4","gaia_2_0","gaia_2_1","gaia_2_2"]`.